app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,40 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
+        <ui-select-match placeholder="Select a storage class">
+          <span>
+            {{$select.selected.metadata.name}}
+          </span>
+        </ui-select-match>
+        <ui-select-choices
+          repeat="sclass as sclass in storageClasses | orderBy: 'metadata.name'">
```

Comment:
We should enable type ahead filtering. Here's an example:

https://github.com/spadgett/origin-web-console/blob/8fd9c66ca45a40d134a941448b1718d80b998f3a/app/views/modals/link-service.html#L10-L18